datacube_alchemist/worker.py

```diff
@@ -257,16 +257,16 @@ def find_fill_missing(self, queue, dryrun):
                 on sub_d.id = sub_s.dataset_ref
                 join agdc.dataset_type as sub_t
                 on sub_d.dataset_type_ref = sub_t.id
-                where sub_t.name = %{output_product}s
+                where sub_t.name = '{output_product}'
```

Comment:
How was this failing?

I prefer leaving correct escaping up to someone else's code, it's too easy to introduce silly bugs, even ignoring SQL injection vulns.

Reply:
I'm not sure how to pass in variables properly. 

The error was:
```
Traceback (most recent call last):
  File "/env/bin/datacube-alchemist", line 8, in <module>
    sys.exit(cli_with_envvar_handling())
  File "/env/lib/python3.6/site-packages/datacube_alchemist/cli.py", line 66, in cli_with_envvar_handling
    cli(auto_envvar_prefix="ALCHEMIST")
  File "/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/env/lib/python3.6/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/env/lib/python3.6/site-packages/datacube_alchemist/cli.py", line 191, in add_missing_to_queue
    n_messages = alchemist.find_fill_missing(queue, dryrun)
  File "/env/lib/python3.6/site-packages/datacube_alchemist/worker.py", line 300, in find_fill_missing
    query, dict(input_products=input_products, output_product=output_product)
TypeError: 'dict' object does not support indexing
```